docs/source/gettingstarted.rst

```diff
@@ -179,6 +179,7 @@ Adding Arguments to write_xlsx()
 
 There are a few arguments you can add to the ``write_xlsx`` function to modify the ``.xlsx`` output file.
 They are:
+- ``max_y``
```

Comment:
Need to have a blank line before and after lists or sometimes sphinx won't render them correctly